### DISCUSSION TITLE
idea

### DISCUSSION THREAD

**Discussion by Mohiit70** [KenyaFlutterDev/flutter_ke#15] [Ideas]
Hey , just sharing some thoughts to enhance our presence

You can make Discord Server, Twitter account for more engaging discussions and Engagement for your organization.

**Comment by mboyamike** [comment]
Thanks. I really appreciate this

I added the twitter account to your pull request yesterday. I'll talk with the main organisers about the discord one

**Comment by Mohiit70** [comment]
> Thanks. I really appreciate this
> 
> I added the twitter account to your pull request yesterday. I'll talk with the main organisers about the discord one

@mboyamike you can check my #14 pr if any suggestion

**Comment by mboyamike** [comment]
Yeah, just suggested a few changes. Just a few additions
